### PR TITLE
Hair - Mickey inspired style

### PR DESCRIPTION
**Pull request by nasehim7**
**Description**
Added Mickey Inspired Hairstyle to avatar creator.

**Checklist**
- [x] linter status: 100% pass
- [x] changes don't break existing behaviour
- [x] commit message follows commit guidelines

**Affected core subsystem(s)**
avatar

**Testing**
Tested locally, screenshots attached:

- [x] Mickey Inspired hairstyle
image

image

**Refers/Fixes**
Refs: #3195

### CODE REVIEW

File: app/avatar/utils.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def get_avatar_context():
                 ['Masks-guy-fawkes'], ['Masks-jack-o-lantern-lighted'], ['Masks-wolverine_inspired'],
                 ['Masks-captain_inspired'], ['Extras-Parrot'], ['Extras-wonderwoman_inspired'],
                 ['Extras-santa_inspired'], ['Extras-reindeer'], ['Masks-gitcoinbot'],
-                ['Masks-batman_inspired'], ['Masks-eye-patch'], ['Masks-darth_inspired'],
+                ['Extras-tattoo'], ['Masks-batman_inspired'], ['Masks-eye-patch'],
```
Comment (thelostone-mc) [inline app/avatar/utils.py]: @nasehim7 why was `Masks-darth_inspired` removed ?

Comment (nasehim7) [reply app/avatar/utils.py]: @thelostone-mc looks like I messed up with my branch :/

Comment (nasehim7) [reply app/avatar/utils.py]: I better be removing this inconsistency by providing a quick fix just wait

Comment (nasehim7) [reply app/avatar/utils.py]: https://github.com/gitcoinco/web/pull/3266

Comment (thelostone-mc) [reply app/avatar/utils.py]: Ah thanks man :)

**Pull request merged**